src/main/resources/users/list.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>List of users</title>
+</head>
+<body>
+
+</body>
+</html>
```

Comment:
Please add a new line at the end of the file

Reply:
nope